dagger/dagger.nim

```diff
@@ -62,10 +61,16 @@ proc new*(T: type DaggerServer, config: DaggerConf): T =
     .withTcpTransport({ServerFlags.ReuseAddr})
     .build()
 
+  let cache = block:
```

Comment:
You don't need the `block`, `if` can be used as an expression as well. So 

```nim
let cache = if config.cacheSize > 0:
      CacheStore.new(cacheSize = config.cacheSize * MiB)
    else:
      CacheStore.new()
```

Works just as well.

Reply:
I find the block easier to read personally, but will update it.

Reply:
Removed the `block:`.